src/Nix/Context.hs

```diff
@@ -20,16 +20,16 @@ data Context m t = Context
     }
 
 instance Has (Context m t) (Scopes m t) where
-  hasLens f (Context x y z w) = (\x' -> Context x' y z w) <$> f x
+  hasLens f a = (\x -> a { scopes = x }) `fmap` f (scopes a)
```

Comment:
why not `<$>`?

Reply:
Because it is pasted from generateLenses :D.

Reply:
Lerned just yesterday that HLS now has the `lsp-lense-show` feature that allows to peek into the lense generated code.